superset-frontend/src/views/CRUD/utils.tsx

```diff
@@ -164,7 +164,7 @@ export function createErrorHandler(handleErrorFunc: (errMsg?: string) => void) {
   return async (e: SupersetClientResponse | string) => {
     const parsedError = await getClientErrorObject(e);
     logging.error(e);
-    handleErrorFunc(parsedError.message || parsedError.error);
+    handleErrorFunc((parsedError.message || parsedError.error) as string);
```

Comment:
any way we can avoid the coercion here?

Reply:
This is likely due to the new type `message?: string | Record<string, any>;` above. We should probably handle the case for `Record<string, any>`. `JSON.stringify`? or perhaps we can find an elegant way to format these records into a human readable string.

Reply:
So maybe we should do:
```typescript
(
  typeof parsedError.message === "string"
    ? parsedError.message
    : parsedError.message
    ? JSON.stringify(parsedError.message)
    : parsedError.error
)
```

Does that make sense/fix the issue?